htmlbasic.go

```diff
@@ -188,6 +188,12 @@ func (html *HTMLBasicType) Write(lineHt float64, htmlStr string) {
 			case "center":
 				html.pdf.Ln(lineHt)
 				alignStr = "C"
+			case "right":
```

Comment:
Adding support for 'right' and 'left' opening html tags. Mapped the right tag with the 'R' alignment of the pdf.WriteAligned function.